azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -76,10 +78,13 @@
   public ContainerCleanupManager(final Props azkProps, final ExecutorLoader executorLoader,
       final ContainerizedImpl containerizedImpl,
       final ContainerizedDispatchManager containerizedDispatchManager) {
-    this.cleanupIntervalMin = azkProps
+    this.executionCleanupIntervalMin = azkProps
         .getLong(
             ContainerizedDispatchManagerProperties.CONTAINERIZED_STALE_EXECUTION_CLEANUP_INTERVAL_MIN,
-            DEFAULT_STALE_CONTAINER_CLEANUP_INTERVAL.toMinutes());
+            DEFAULT_STALE_EXECUTION_CLEANUP_INTERVAL.toMinutes());
+    this.containerCleanupIntervalMin = azkProps.getLong(
+        ContainerizedDispatchManagerProperties.CONTAINERIZED_STALE_CONTAINER_CLEANUP_INTERVAL_MIN
+        , DEFAULT_STALE_CONTAINER_CLEANUP_INTERVAL.toMinutes());
```

Comment:
extreme nit: keep the comma in line above.

Reply:
Made the change